content/docs/start/data-and-model-versioning.md

```diff
@@ -212,8 +208,8 @@ $ dvc push
 
 ## Switching between versions
 
-The regular workflow is to use `git checkout` first to switch a branch, checkout
-a commit, or a revision of a `.dvc` file, and then run `dvc checkout` to sync
+The regular workflow is to use `git checkout` first (to switch a branch or
+checkout a commit/revision of a `.dvc` file) and then run `dvc checkout` to sync
```

Comment:
Prob better, yes 👍 Though the parenthesis is still kind of long. Consider completely rewriting. For now:
```suggestion
The regular workflow is to use `git checkout` first (to switch a branch or
checkout a commit/revision of a `.dvc` file), and then run `dvc checkout` to sync
```

Reply:
I guess this comma wasn't needed, resolving.